front/lib/resources/data_source_resource.ts

```diff
@@ -257,6 +272,12 @@ export class DataSourceResource extends ResourceWithVault<DataSource> {
     return getDataSourceUsage({ auth, dataSource: this.toJSON() });
   }
 
+  // sId logic.
+
+  get sId(): string {
```

Comment:
Let's not play that game and be explicit with the fact that we use the name when we use the name. Otherwise it'll make a refactor to sId harder rather than easier I think.

Reply:
I thought the way around but anyway will use `name`.

Reply:
Well my thinking goes as follow: if I introduce sId() to be a sqids now I have to go back to call sites of that new sId to replace with .name which feels risky out of context.